sql/mozfun/norm/browser_version_info/udf.sql

```diff
@@ -0,0 +1,76 @@
+CREATE OR REPLACE FUNCTION norm.browser_version_info(version_string STRING)
+RETURNS STRUCT<
+  version STRING,
+  major_version NUMERIC,
+  minor_version NUMERIC,
+  is_major_release BOOLEAN
+> AS (
+  STRUCT(
+    version_string AS version,
+    norm.truncate_version(version_string, 'major') AS major_version,
```

Comment:
I'd say `_version` could be omitted from the field name since it's already part of the `version` strict. `version.major` vs `version.major_version` doesn't add much in terms of readability IMHO.